### ISSUE TITLE
root_dist in postProcessing returns unexpected values

### ISSUE THREAD

**Issue by hcOnel** [turbinesFoam/turbinesFoam#259]
In the blade element files in the directory `postProcessing/actuatorLineElements/0/`, there is a variable called `root_dist`. My first assumption is that root_dist is a value between 0 and 1, corresponding to the rational position of the element center along blade span. It remains constant for individual blade element throughout time steps.
However, in my case, the `root_dist` for elements is in order:
```
   0.034827
   0.089249
   0.132806
   0.182896
   0.235151
   0.287407
   0.339663
   0.386045
   0.437429
   0.496431
   0.548687
   0.600942
   0.651032
   0.694589
   0.731582
   0.762043
   0.779473
```
despite that rotor radius is 63 and my last geometric point is at radius=63.
Attached is my fvOptions file. Could not figure out why. What do I understand wrong?

[fvOptions.txt](https://github.com/turbinesFoam/turbinesFoam/files/2177135/fvOptions.txt)

**Comment by hcOnel** [comment]
It is caused by giving azimuthal degrees to some elements. I thought that parameters like azimuth and axialDistance is used to locate the chordMount point but I guess they also rotate the elements.

**Comment by petebachant** [comment]
Interesting question. I've never tested a turbine with elements having an axial distance other than zero, e.g., a HAWT with coned blades. I'll need to look back through the code, but the distance may be a straight line distance between the root geometry point and the element quarter chord, which would explain the values less than one for a heavily coned or twisted (azimuthally) blade.

**Comment by hcOnel** [comment]
Yes, I was playing with geometry parameters to see their behavior and set azimuth or axialDistance value of a single section to a large value like 20 or 45deg. That caused the final radius to be 0.77ish. A confusing way of defining the geometry I suppose.
